recipes/hazelcastcxx/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class HazelcastCxx(ConanFile):
+    name = "hazelcastcxx"
+    description = "Hazelcast is C++ API for in memory database."
```

Comment:
Would changing the description to something like `C++ client library for Hazelcast in-memory database` read better ?

Reply:
I was thinking that we are trying to use `API` instead of `client` and hence preferred the other description but I am putting it back as suggested.